### ISSUE TITLE
sympy requires antlr 4.11

### ISSUE THREAD

**Issue by bhpayne** [allofphysicsgraph/proofofconcept#287]
Inside the container,
```python
# python3
Python 3.8.10 (default, Nov 22 2023, 10:22:35) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sympy
>>> sympy.__version__
'1.12'
>>> from sympy.parsing.latex import parse_latex
>>> x = parse_latex("a = b")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/dist-packages/sympy/parsing/latex/__init__.py", line 35, in parse_latex
    return _latex.parse_latex(s)
  File "/usr/local/lib/python3.8/dist-packages/sympy/parsing/latex/_parse_latex_antlr.py", line 66, in parse_latex
    raise ImportError("LaTeX parsing requires the antlr4 Python package,"
ImportError: LaTeX parsing requires the antlr4 Python package, provided by pip (antlr4-python3-runtime) or conda (antlr-python-runtime), version 4.11
>>> exit()
root@b8dadce452c7:/code# pip3 install antlr4-python3-runtime

Requirement already satisfied: antlr4-python3-runtime in /usr/local/lib/python3.8/dist-packages (4.13.1)
```

Looks like I need to pin the pip antlr install to 4.11 (which was released Sept 2022)

**Comment by bhpayne** [comment]
https://pypi.org/project/antlr4-python3-runtime/#history

**Comment by bhpayne** [comment]
Solved in commit 7278062412fc7f4a920df77723d40a05a09b68d5
